specification/resources/resource-manager/readme.md

```diff
@@ -82,17 +82,31 @@ tag: package-snapshots-2022-11
 ```
 
 ``` yaml $(package-bicep)
-tag: package-bicep-2023-11
+tag: package-2024-04
 ```
 
+
+### Tag: package-2024-04
+
+These settings apply only when `--tag=package-2024-04` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-04'
+input-file:
+  - Microsoft.Authorization/stable/2024-04-01/policyAssignments.json
+  - Microsoft.Authorization/stable/2024-04-01/policyDefinitionVersions.json
+  - Microsoft.Authorization/stable/2024-04-01/policyDefinitions.json
+  - Microsoft.Authorization/stable/2024-04-01/policySetDefinitionVersions.json
+  - Microsoft.Authorization/stable/2024-04-01/policySetDefinitions.json
```

Comment:
pl add these files to that folder

Reply:
This PR will update just the assignment. Removed the entries from the list here